code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -134,6 +134,30 @@ obj/item/weapon/gun/energy/retro
 	damage_multiplier = 1
 	charge_cost = 100
 
+/obj/item/weapon/gun/energy/makeshiftlaser
+	name = "Makeshift laser carbine"
+	desc = "A Makeshift laser carbine, rather wastefull on its chage, but nonetheless reliable"
+	icon = 'icons/obj/guns/energy/laser.dmi'
+	icon_state = "laser"
+	item_state = "laser"
+	item_charge_meter = TRUE
+	fire_sound = 'sound/weapons/Laser.ogg'
+	slot_flags = SLOT_BELT|SLOT_BACK
+	w_class = ITEM_SIZE_NORMAL
+	force = WEAPON_FORCE_NORMAL
+	origin_tech = list(TECH_COMBAT = 2, TECH_MAGNET = 1)
+	matter = list(MATERIAL_STEEL = 20, MATERIAL_PLASTIC = 15)
+	zoom_factor = 0
+	damage_multiplier = 1 //worst lightfall
+	charge_cost = 75 //ditto
```

Comment:
This is better than the Cog, a "real" gun. You might want to bump the charge cost up or reduce the damage. For reference the Cog has 1 dmg mult and 100 charge cost

Reply:
well making it worst in both ways